packages/cursorless-org-docs/docusaurus.config.mts

```diff
@@ -43,23 +48,40 @@ function remarkPluginFixLinksToRepositoryArtifacts(): Transformer<Root> {
       );
       const artifact = resolve(file.dirname!, url);
       const artifactRelative = relative(repoRoot, artifact).replace(/\\/g, "/");
+      const fileRelative = relative(repoRoot, file.path).replace(/\\/g, "/");
 
-      // We host all files under docs, will resolve as a relative link
-      if (
-        artifactRelative.startsWith("packages/cursorless-org-docs/src/docs/")
-      ) {
+      // We host all files under docs. Will resolve as a relative link, but
+      // relative links pointing to a folder passing between user and
+      // contributing are not resolved correctly by docusaurus so we need to
+      // rewrite them.
+      if (artifactRelative.startsWith(docsRelative)) {
+        if (
+          isFolder(url) &&
+          passingBetweenUserAndContributing(fileRelative, artifactRelative)
+        ) {
+          node.url = "/docs/" + artifactRelative.slice(docsRelative.length);
```

Comment:
I have no idea why this is required. Linking to a markdown file between contributing and user works fine, but not a folder. `../user/languages` incorrectly got resolved as `/docs/contributing/user/languages`. This problem does not occur with files.

Reply:
Could have something to do with this? But I'm not sure why files work and not folders.

https://docusaurus.io/docs/markdown-features/links#:~:text=Markdown%20file%20references,use%20URL%20links.